asab/log.py

```diff
@@ -446,13 +446,23 @@ def _on_write(self):
 		self._write_ready = True
 		self._loop.remove_writer(self._socket)
 
+		error_counter = 0
 		while not self._queue.empty():
 			msg = self._queue.get_nowait()
 			try:
 				self._socket.sendall(msg)
 			except Exception as e:
-				print("Error when writing to syslog '{}'".format(self._address), e, file=sys.stderr)
+				if error_counter > 100:
```

Comment:
there should not be any counter and exit. simply print the error and the socket error will hopefully fix itself in a bit

Reply:
also do not `self._enqueue(msg)`
the msg should be thrown away